source/switching_to_live/before_you_switch_to_live/index.html.md.erb

```diff
@@ -44,24 +50,43 @@ to GOV.UK Pay.
 
 ## Request a live GOV.UK Pay account
 
-1. Sign in to your [GOV.UK Pay
-   account](https://selfservice.payments.service.gov.uk/login).
+Requesting a live account is the same regardless of whether you are a Stripe or non-Stripe user.
 
-1. Select __My services__.
+1. Go to the [Request a live account](https://pay-admin.cloudapps.digital/request-to-go-live) page on GOV.UK Pay and select __Start now__.
+
+1. Enter the name of your organisation and select __Continue__.
+
+1. Choose how you will process payments and select __Continue__.
+
+1. Select the __Accept legal terms__ checkbox and select __Continue__.
+
+You will see the __Request submitted__ page. GOV.UK Pay will email you a link to your live account within one working day.
+
+When your account is live, it will appear in the __My services__ page labelled as a __Live card account__.
+
+## Complete your information
+
+The information you provide differs depending on whether you use Stripe or another PSP.
+
+### Stripe
+
+GOV.UK Pay will contact you to verify your identity. You must provide GOV.UK Pay with your:
 
-1. For the test account you want to make live, select __Manage team members__
-   on the dashboard.
+- bank sort code and account number
+- responsible person's name, date of birth and home address
```

Comment:
might be worth explaining this a little more - e.g. a person who is authorised to sign contracts on behalf of the organisation

Reply:
agree